capsules/src/at_commands_driver.rs

```diff
@@ -0,0 +1,237 @@
+//! Userspace driver for AT commands.
+//!
+//! This driver was implemented for the ESP32 using AT commands.
```

Comment:
If it's specific to the ESP32, it ought to be named something for specific. If it's not actually specific to the ESP32.

In both cases, the comment/description should explain a bit more what the functionality is. E.g., what interface does a process get?

Reply:
Ok I updated the comment to hopefully clarify that this should be generic, and to explain a little about how it is designed to be used.